### PR TITLE
refactor: removed unused import from the provider file

### PR DESCRIPTION
**Pull request by alkavats1**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

**Comment by alkavats1** [comment]
Hi @AndrewKushnir please review the PR
Thanks :)

File: packages/common/http/testing/src/provider.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@
  * found in the LICENSE file at https://angular.io/license
  */
 
-import {HttpBackend, HttpClientModule} from '@angular/common/http';
+import {HttpBackend} from '@angular/common/http';
 import {NgModule, Provider} from '@angular/core';
```
Comment (AndrewKushnir) [inline packages/common/http/testing/src/provider.ts:10]: Could you please also remove the `NgModule` symbol import (it's also unused)?
```suggestion
import {Provider} from '@angular/core';
```

**Comment by alkavats1** [comment]
Hi @AndrewKushnir, I have changed the code, according to your suggestion, please review and merge
Thanks :)

**Review by AndrewKushnir — APPROVED** [review]
@alkavats1 thanks for the refactoring!

**Comment by AndrewKushnir** [comment]
**Caretaker note**: low risk change, no presubmit is required.

**Comment by AndrewKushnir** [comment]
This PR was merged into the repository by commit 66824b77154fb0fd0a4d1ffc35976cb404ec3226.

**Pull request closed without merging**